util/stream.go

```diff
@@ -0,0 +1,19 @@
+package util
+
+import "crypto/cipher"
+
+// ConstantStream is a cipher.Stream which always returns
+// the same value.
+type ConstantStream struct {
```

Comment:
Because since users are going to have a reference on it, they may have to understand what's it about and for the doc to show the comments it must be public ... ?